ci/build/copy_build_artifacts.sh

```diff
@@ -33,8 +33,8 @@ if [[ "$OSTYPE" == "msys" ]]; then
   ARTIFACT_MOUNT="/c/artifact-mount"
 fi
 
-export PATH=/opt/python/cp38-cp38/bin:$PATH
-pip install -q -c python/requirements_compiled.txt aws_requests_auth boto3 pyopenssl
+export PATH=/opt/python/cp39-cp39/bin:$PATH
```

Comment:
`/opt/python/cp39-cp39/` is the python path in manylinux docker image where this script is also invoked; 38 works fine for now since it's still the old manylinux image but should be upgrade to 39 too